### ISSUE TITLE
Should color mode be a kwarg arg to Points?

### ISSUE THREAD

**Issue by sofroniewn** [napari/napari#931]
## ❓ Questions

Should color mode be a kwarg arg to Points (i.e. `edge_color_mode` and `face_color_mode`)? The reason for this is that these properties will affect how things get rendered on the screen and so if we want to enable full round trips of our data through serialization and back again to have the exact same thing appear on the screen they need to be included. Thoughts @kevinyamauchi?

**Comment by kevinyamauchi** [comment]
Yeah that makes sense to me. My only hesitation is that I generally try to avoid kwargs that can conflict (e.g., `Points(edge_color_mode='cycle', edge_color = [[1, 0, 0, 1]])`). This one isn't too bad and should be easy to check for, so I think we can give helpful error messages.

I can work on it soon. Although, think this would be a good first issue if anybody else wants to take a crack at it. In short, one would have to:

- add the `face_color_mode` and `edge_color_mode` to [`Points.__init__()`](https://github.com/napari/napari/blob/7a04984472db9a0b74c4efc26d56a6f56b4ce9dd/napari/layers/points/points.py#L207)
- add `face_color_mode` and `edge_color_mode` to [`Points._get_state()`](https://github.com/napari/napari/blob/7a04984472db9a0b74c4efc26d56a6f56b4ce9dd/napari/layers/points/points.py#L964) (serialization function)
- update the [add_points()](https://github.com/napari/napari/blob/7a04984472db9a0b74c4efc26d56a6f56b4ce9dd/napari/components/add_layers_mixin.py#L241) and [view_points()](https://github.com/napari/napari/blob/7a04984472db9a0b74c4efc26d56a6f56b4ce9dd/napari/view_layers.py#L142) signatures to pass `face_color_mode` and `edge_color_mode`.
- update docstrings

**Comment by neuromusic** [comment]
ok, I took a stab at this (see #955), but tests are currently failing. I'll try to swing back to this later
